python/pip_install/tools/dependency_resolver/__init__.py

```diff
@@ -11,4 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
Nit pick.  Please remove the file cleanup in this PR.  I would like to not have extra needed files in the commit.